inji-verify-sdk/src/components/qrcode-verification/QRCodeVerification.tsx

```diff
@@ -358,7 +358,8 @@ const QRCodeVerification: React.FC<QRCodeVerificationProps> = ({
         clientId,
         transactionId ?? undefined,
         presentationDefinitionId,
-        presentationDefinition
+        presentationDefinition,
+        true // default to true for data-share vc
```

Comment:
This is not correct, it should default to false. From UI component we can send as true as required. But in SDK you should not default to true.